net.audiorelay.AudioRelay.yaml

```diff
@@ -0,0 +1,54 @@
+app-id: net.audiorelay.AudioRelay
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+command: AudioRelay
+separate-locales: false
+
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --socket=pulseaudio
+  - --device=dri
+
+modules:
+  - name: audiorelay
+    buildsystem: simple
+    build-commands:
+      - install -Dm755 audiorelay.sh /app/bin/AudioRelay
+      - install -Dm755 apply_extra -t /app/bin
+      - install -Dm644 net.audiorelay.AudioRelay.metainfo.xml -t /app/share/metainfo
+      - install -Dm644 net.audiorelay.AudioRelay.png -t /app/share/icons/hicolor/512x512/apps
+      - install -Dm644 net.audiorelay.AudioRelay.desktop -t /app/share/applications
+    sources:
+      - type: script
+        commands:
+          - exec /app/extra/bin/AudioRelay "$@"
+        dest-filename: audiorelay.sh
+
+      - type: script
+        commands:
+          - tar -xvf audiorelay.tar.gz
+          - rm -rf audiorelay.tar.gz
+        dest-filename: apply_extra
+
+      - type: file
+        path: net.audiorelay.AudioRelay.png
+
+      - type: file
+        path: net.audiorelay.AudioRelay.desktop
+
+      - type: file
+        path: net.audiorelay.AudioRelay.metainfo.xml
+
+      - type: extra-data
+        filename: audiorelay.tar.gz
+        url: https://dl.audiorelay.net/setups/linux/audiorelay-0.27.5.tar.gz
```

Comment:
so this is a universal binary that runs on both x86_64 and aarch64?

Reply:
No, it only works on x86_64. sorry I forget to add limits